client/client-multi/client-cpp/impl/channel/connectivity/PingConnectivityChecker.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2014 CyberVision, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "kaa/channel/connectivity/PingConnectivityChecker.hpp"
+
+#include <string>
+#include <sstream>
+#include <cstdint>
+
+#include <boost/asio/ip/tcp.hpp>
+#include <boost/asio/connect.hpp>
+#include <boost/asio/io_service.hpp>
+
+#include "kaa/logging/Log.hpp"
+
+namespace kaa {
+
+const std::string PingConnectivityChecker::DEFAULT_HOST = "www.google.com";
```

Comment:
It was made only for compatibility with Java SDK.
Insied C++ SDK uses its own connectivity checker than pings one of Kaa Bootstrap server.
See details in https://github.com/kaaproject/kaa/blob/master/client/client-multi/client-cpp/impl/channel/connectivity/IPConnectivityChecker.cpp